.all-contributorsrc

```diff
@@ -236,11 +236,25 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "sunnytsang1998",
+      "name": "sunnytsang1998",
+      "avatar_url": "https://avatars.githubusercontent.com/u/207208443?v=4",
+      "profile": "https://github.com/sunnytsang1998",
+      "contributions": [
+        "bug"
+      ]
     }
   ],
   "contributorsSortAlphabetically": true,
   "projectName": "eslint-plugin-package-json",
   "projectOwner": "JoshuaKGoldberg",
   "repoType": "github",
-  "commitConvention": "angular"
+  "commitConvention": "angular",
+  "files": [
```

Comment:
@JoshuaKGoldberg it looks like this came back, which I believe the CTA update PR removed.  FYI

Reply:
I'm so tired of dealing with all-contributors edge cases 🫠 

https://github.com/JoshuaKGoldberg/create-typescript-app/issues/2203 -> https://github.com/JoshuaKGoldberg/create-typescript-app/pull/2204